### PR TITLE
CONTRIBUTING: make more Linuxbrew-specific.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Linuxbrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Linuxbrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

-----

Remove things that incorrectly point people towards Homebrew.

Replaces broken https://github.com/Linuxbrew/brew/pull/427.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-# Contributing to Homebrew
-First time contributing to Homebrew? Read our [Code of Conduct](https://github.com/Homebrew/brew/blob/master/CODEOFCONDUCT.md#code-of-conduct).
+# Contributing to Linuxbrew
+First time contributing to Linuxbrew? Read our [Code of Conduct](https://github.com/Linuxbrew/brew/blob/master/CODEOFCONDUCT.md#code-of-conduct).
 
 ### Report a bug
 
 * run `brew update` (twice)
 * run and read `brew doctor`
-* read [the Troubleshooting Checklist](http://docs.brew.sh/Troubleshooting.html)
```
Comment (bw-yuhogg) [inline CONTRIBUTING.md:14]: Can we point to https://github.com/Linuxbrew/brew/blob/master/docs/Troubleshooting.md instead of simply deleting this line?

Comment (maxim-belkin) [reply CONTRIBUTING.md:14]: yes we can

Comment (maxim-belkin) [reply CONTRIBUTING.md:14]: I suggest editing the PR before committing

Comment (MikeMcQuaid) [reply CONTRIBUTING.md:14]: Done.

**Review by bw-yuhogg — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
🚢?

**Comment by bw-yuhogg** [comment]
> 🚢?

Just waiting for @maxim-belkin's approval.

**Review by maxim-belkin — APPROVED** [review]

**Pull request merged**